aha-services.gemspec

```diff
@@ -31,9 +31,6 @@ Gem::Specification.new do |s|
   # For Trello integration.
   s.add_dependency "reverse_markdown"
 
-  # For Redmine integration.
-  s.add_dependency "ClothRed"
```

Comment:
It was moved to the Gemfile proper (see top file on this page). You can't specify a Github gem source within the gemspec.